src/components/Card/Card.css

```diff
@@ -88,14 +94,14 @@
 .card .channels .channel-inner {
   display: flex;
   justify-content: space-between;
-  padding-bottom: 10px;
   align-items: flex-end;
   border-top: 1px solid #ccc;
-  padding-top: 10px;
+  padding: 10px;
```

Comment:
This should be `padding: 10px 0px` or you will also add left and right padding

Reply:
That's exactly what I meant by adding padding for all sides. If you don't add left and right padding you will end up with text touching the sides which isn't readable as shown in the screenshot below.

![card-description-example](https://user-images.githubusercontent.com/7634871/54832723-fdf89480-4cbc-11e9-9940-e6121bb14cb9.png)